internal/verifier/verification_task.go

```diff
@@ -59,11 +59,19 @@ type VerificationTask struct {
 	Type       verificationTaskType   `bson:"type"`
 	Status     verificationTaskStatus `bson:"status"`
 	Generation int                    `bson:"generation"`
-	Ids        []interface{}          `bson:"_ids"`
+
+	// For failed tasks, this stores the document IDs missing on
+	// one cluster or the other.
+	Ids []interface{} `bson:"_ids"`
```

Comment:
nit: Maybe rename this to `MissingIDs`? Also note that Go style is to use all for abbreviations like "ID", as opposed to "Id".

Reply:
Unfortunately this gets used for other purposes in other event types. I’ll rename `Ids` as you suggest.